request_a_govuk_domain/request/templates/session_ended.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+{% load static %}
+{% load crispy_forms_tags crispy_forms_gds %}
+{% load govuk_frontend_django %}
+{% load csp %}
+
+{% block title %}Session Ended{% endblock %}
+{% block session_ended %}{% endblock %} <!-- we do not show the model dialog on session-ended page, so making it empty -->
+
+{% block content %}
+{% load custom_filters %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        {% error_summary form %}
```

Comment:
Why is this a form. Can the button just link to the start page (if that resets the session)?